src/EditCell.tsx

```diff
@@ -1,37 +1,38 @@
 import { useState, useCallback } from 'react';
+import { createPortal } from 'react-dom';
 import { css } from '@linaria/core';
 
-import EditorContainer from './editors/EditorContainer';
+import { useMouseDownOutside } from './hooks';
 import { getCellStyle, getCellClassname } from './utils';
-import type { CellRendererProps, SharedEditorProps, Omit } from './types';
+import type { CellRendererProps, SharedEditorProps } from './types';
 
 const cellEditing = css`
   padding: 0;
 `;
 
-const cellEditingClassname = `rdg-cell-editing ${cellEditing}`;
+// TODO: remove 1 of the two classes
+const cellEditingClassname = `rdg-cell-editing rdg-editor-container ${cellEditing}`;
 
 type SharedCellRendererProps<R, SR> = Pick<
   CellRendererProps<R, SR>,
   'rowIdx' | 'row' | 'column' | 'colSpan'
 >;
 
-interface EditCellProps<R, SR>
-  extends SharedCellRendererProps<R, SR>,
-    Omit<React.HTMLAttributes<HTMLDivElement>, 'style' | 'children'> {
+interface EditCellProps<R, SR> extends SharedCellRendererProps<R, SR> {
   editorProps: SharedEditorProps<R>;
+  onKeyDown: Required<React.HTMLAttributes<HTMLDivElement>>['onKeyDown'];
 }
 
 export default function EditCell<R, SR>({
-  className,
   column,
   colSpan,
   row,
   rowIdx,
-  editorProps,
-  ...props
+  onKeyDown,
+  editorProps
 }: EditCellProps<R, SR>) {
   const [dimensions, setDimensions] = useState<{ left: number; top: number } | null>(null);
```

Comment:
Really wonder if we still need this

Reply:
I think this was needed when we were rendering the whole editor in a portal. I am okay with removing them add them back if needed.